hack/build/operator/build

```diff
@@ -27,11 +27,9 @@ gitHash="github.com/coreos/etcd-operator/version.GitSHA=${GIT_SHA}"
 
 go_ldflags="-X ${ldKVPairs} -X ${gitHash}"
 
-GOBIN=${bin_dir} GOOS=linux GOARCH=amd64 CGO_ENABLED=0 go install -installsuffix cgo -ldflags "$go_ldflags" ./cmd/operator/
-mv ${bin_dir}/operator ${bin_dir}/etcd-operator
+GOOS=linux GOARCH=amd64 CGO_ENABLED=0 go build -o ${bin_dir}/etcd-operator -installsuffix cgo -ldflags "$go_ldflags" ./cmd/operator/
```

Comment:
This looks better.

I would also want to keep the `go install` method for local development because of faster builds. Can you make it optional:

```
if [ "$GOINSTALL" == "y" ]
then
	GOBIN=${bin_dir} GOOS=linux GOARCH=amd64 CGO_ENABLED=0 go install ...
else
	GOOS=linux GOARCH=amd64 CGO_ENABLED=0 go build ...
fi
```